docs/bindings.md

```diff
@@ -0,0 +1,116 @@
+# Bindings
+
+Bindings bridge the `superchain-registry` to other languages with associated types.
+
+## Rust Bindings
+
+There are two rust [crates](crates.io) that expose rust bindings.
+- [`bindings/rust-primitives`][rp]
+- [`bindings/rust-bindings`][rb]
+
+### Rust Primitives
+
+[`bindings/rust-primitives`][rp] contains rust types for the objects defined in the `superchain-registry`.
+The `rust-primitives` directory is a `no_std` [crate][rpc] called [`superchain-primitives`][sp].
+It defines the `std` and `serde` feature flags that enable `std` library use and `serde` serialization
+and deserialization support, respectively. Enabling these feature flags on the crate are as simple as
+adding the `features` key to the dependency declaration. For example, in a project's `Cargo.toml`, to
+use the `superchain-primitives` crate with `serde` support, add the following to the `[dependencies]`
+section.
+
+```toml
+# By default, superchain-primitives enables the `std` and `serde` feature flags.
+superchain-primitives = { version = "0.2", features = [ "serde" ], default-features = false }
+```
+
+[`superchain-primitives`][sp] has minimal dependencies itself and uses [alloy][alloy] Ethereum
+types internally. Below provides a breakdown of some core types defined in [`superchain-primitives`][sp].
+
+**`ChainConfig`**
+
+The [`ChainConfig`][cc] is the chain configuration that is output from the `add-chain` command in the
+`superchain-registry`. It contains genesis data, addresses for the onchain system config, hardfork
+timestamps, rpc information, and other superchain-specific information. Static chain config files
+are defined in [../superchain/configs/](../superchain/configs/).
+
+**`RollupConfig`**
+
+The [`RollupConfig`][rc], often confused with the `ChainConfig`, defines the configuration for
+a rollup node. The [`RollupConfig`][rc] defines the parameters used for deriving the L2 chain as
+well as batch-posting data on the flip side. It effectively defines the rules for the chain used by
+consensus and execution clients. (e.g. `op-node`, `op-reth`, `op-geth`, ...)
+
+[`superchain-primitives`][sp] also exposes a few default `RollupConfig`s for convenience, providing an alternative to depending on [`superchain-registry`][sr] with `serde` required.
+
+**`Superchain`**
+
+[`Superchain`][s] defines a superchain for a given L1 network. It holds metadata such as the
+name of the superchain, the L1 anchor information (chain id, rpc, explorer), and default
+hardfork configuration values. Within the [`Superchain`][s], there's a list of [`ChainConfig`][cc]s
+that belong to this superchain.
+
+### Rust Bindings
+
+[`bindings/rust-bindings`][rb] exports rust type definitions for chains in the `superchain-registry`.
+The `rust-bindings` directory is a `no_std` [crate][rbc] called [`superchain-registry`][src].
```

Comment:
This line isn't rendering properly for me I just see [`superchain-registry`][src] ?